05. Assignment-5/src/Components/CardComponent.js

```diff
@@ -5,7 +5,7 @@ const CardComponent = (props) => {
     const { img, name, role } = props.userInfo;
     return (        
       <div className="col-sm-6 col-md-4 col-lg-3 col-xl-3 mt-4" key={props.id}>
-          <div className='d-flex align-items-center p-3 bg-dark rounded-lg flex-column'>
+          <div className='d-flex align-items-center p-3 bg-dark rounded-lg flex-column' style={{ width: "15rem", height: "23rem"}}>
               <div className='person-img'>
                 <img src={img} alt="img" className="img-fluid"></img>
```

Comment:
alt text should be like - {User.name}. Screen Readers read it like {User.name}'s image. So write it like it should be telling info about the image.